### PR TITLE
fix: format on the available and unavailable content on the button

### PR DESCRIPTION
**Pull request by RodrigoTadeuF**
#### How to test it?

[Workspace](https://rfonseca--unileverb2b.myvtex.com/admin/cms/site-editor)

#### Screenshots or example usage:

image
image

#### Describe alternatives you've considered, if any.

I Don't think it have one

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed 
+
+- Add formatedIOMessage on the AddtoCartButton to format the available and the unavailable content and also out the import on the file, this allows the format to urdu texts to be added.
```
Comment (lariciamota) [inline CHANGELOG.md:12]: What do you think about this suggestion?
```suggestion
- Available and unavailable content text wasn't being translated. So, formatedIOMessage was added to AddtoCartButton.
```

**Review by lariciamota — APPROVED** [review]
Looks good code-wise! But I'm not sure how to test it because I don't have much context of this app, so I ask you to wait for more reviews 😄

File: CHANGELOG.md (hunk 1 above)
Comment (RodrigoTadeuF) [reply CHANGELOG.md:12]: I think it is better yes for comprehension, let me change.

File: react/AddToCartButton.tsx [hunk 2]
```diff
@@ -206,14 +207,14 @@ function AddToCartButton(props: Props) {
...
+          event: 'addToCart',
+          items: pixelEventItems,
+        }
         : {
-            event: 'addToCart',
-            items: pixelEventItems,
-          }
+          event: 'addToCart',
+          items: pixelEventItems,
+        }
```
Comment (victorhmp) [inline react/AddToCartButton.tsx:217]: Was this a lint change? 🤔

**Review by victorhmp — APPROVED** [review]
LGTM 👏🏻

File: react/AddToCartButton.tsx (hunk 2 above)
Comment (RodrigoTadeuF) [reply react/AddToCartButton.tsx:217]: Yes

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**